skgstat/Kriging.py

```diff
@@ -30,6 +30,11 @@ class IllMatrixError(RuntimeWarning):
 def inv_solve(a, b):
     return inv(a).dot(b)
 
+def sparse_dok_get(m, fill_value=np.NaN):
```

Comment:
Can we rename it to `_sparse_dok_get`, as this is not intended to be exported from the module, right?
And maybe a one-liner docstring might be helpful

Reply:
Sure. It's just a utility function.